### PR TITLE
feat: initial python starter template

### LINKED ISSUE #8: Python template
We need a new project template that supports python, using the newer infrastructure as code features. This item depends on https://github.com/nitrictech/python-sdk/issues/60

### PR DESCRIPTION
**Pull request by jyecusch**
Closes: https://github.com/nitrictech/templates/issues/8

Tested locally and appears to work fine, but let's wait for the Python SDK changes to be fully tested and published before we move this out of draft.

### CODE REVIEW

**Comment by jyecusch** [comment]
Tested locally and appears to work fine, but let's wait for the Python SDK changes to be fully tested and published before we move this out of draft.

**Comment by tjholm** [comment]
@jyecusch we should add utilities for use with `nitric start`, including hot-reloading for this template.

File: repository.yaml [hunk 1]
```diff
@@ -2,4 +2,6 @@ templates:
   - name: TypeScript - Starter
     path: ./typescript-starter
   - name: JavaScript - Starter
-    path: ./javascript-starter
\ No newline at end of file
+    path: ./javascript-starter
+  - name: Python - Starter (unstable)
```
Comment (tjholm) [inline repository.yaml]: any alternatives to `unstable` we could use?

Comment (HomelessDinosaur) [reply repository.yaml]: Experimental or Preview?

Comment (tjholm) [reply repository.yaml]:
```suggestion
  - name: Python - Starter (preview)
```

**Review by tjholm — APPROVED** [review]

**Pull request merged**